src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Deprecation.php

```diff
@@ -59,6 +60,10 @@ class Deprecation
      */
     public function __construct($message, array $trace, $file)
     {
+        if (isset($trace[2]['class']) && DebugClassLoader::class === $trace[2]['class']) {
```

Comment:
Shouldn't we do it for `LegacyDebugClassLoader` (old class from symfony/debug) too?

Reply:
Yes, I think so.